src/pymor/reductors/neural_network.py

```diff
@@ -160,16 +190,31 @@ def reduce(self, hidden_layers='[(N+P)*3, (N+P)*3]', activation_function=torch.t
             # set parameters for neural network and training
             neural_network_parameters = {'layer_sizes': layer_sizes,
                                          'activation_function': activation_function}
+            if loss_function == 'weighted MSE':
+                if hasattr(self, 'weights'):
+                    weights = self.weights
+
+                    def weighted_mse_loss_function(inputs, targets):
+                        return (weights * (inputs - targets) ** 2).mean()
+
+                    loss_function = weighted_mse_loss_function
+                    self.logger.info('Using weighted MSE loss function ...')
+                else:
+                    self.logger.warn('No weights for weighted MSE loss available. Switching to default loss ...')
```

Comment:
Why a warning and no error?

Reply:
Replaced the warning in [39fb090](https://github.com/pymor/pymor/pull/1559/commits/39fb090673ee4206ee359c66c36e7b5317bff7b7). I think this was a leftover from some testing I performed.